programs/README.md

```diff
@@ -0,0 +1,30 @@
+# Foundation Program RFCs
```

Comment:
does this make sense anymore with https://github.com/nodejs/bootstrap/pull/23?

Should we break out the best bits from this and bring it into that PR?